### PR TITLE
Add note about vsphere cloud provider vCenter folder structure

### PR DESCRIPTION
**Pull request by c3y1huang**
# Describe your changes
Add note about the correct vCenter folder structure with in-tree vSphere cloud provider.

# Related Issues / Projects
Relates to: https://github.com/SUSE/doc-caasp/issues/884

Signed-off-by: Chin-Ya Huang <chin-ya.huang@suse.com>

### CODE REVIEW

File: adoc/deployment-bootstrap.adoc [hunk 1]
```diff
@@ -376,6 +376,15 @@ For clusters provisioned by any other method, ensure virtual machines are set to
 For more information, refer to: link:https://docs.vmware.com/en/VMware-vSphere/6.7/Cloud-Native-Storage/GUID-3501C3F2-7D7C-45E9-B20A-F3F70D1E4679.html[Configure Kubernetes Cluster Virtual Machines] .
 ====
 
+[IMPORTANT]
+====
+All virtual machines must exist in a folder and provide as the `folder` variable in the `vsphere.conf` before bootstrap.
+
+Clusters provisioned following link:{docurl}single-html/caasp-deployment/#_using_terraform[Deploying VMs from the Template] with `cpi_enable = true` automatically create and place all cluster node virtual machines inside a `*-cluster` folder.
+
```
Comment (Martin-Weiss) [inline adoc/deployment-bootstrap.adoc:384]: What is I do NOT want to use a folder and want to have all VMs in the "root"? What is the default I need to enter for "folder"?

Comment (c3y1huang) [reply adoc/deployment-bootstrap.adoc:384]: It is required by the in-tree plugin to look for a folder under `/vm`. Storage class cannot be used with an invalid `folder` variable.

**Comment by nkoranova** [comment]
As far as I can tell this is relevant to v5 as well as v4, is that true @c3y1huang ?

**Comment by c3y1huang** [comment]
> As far as I can tell this is relevant to v5 as well as v4, is that true @c3y1huang ?

Yes, its relevant in both v5 and v4.

**Comment by nkoranova** [comment]
cherry-puicked to maintenance/CaaS4 with https://github.com/SUSE/doc-caasp/commit/393b2f8137101a4be8f6a86f41660a7d687468d9

**Pull request merged**